sunpy/map/sources/tests/test_wispr_source.py

```diff
@@ -0,0 +1,292 @@
+"""Tests for EUI Solar Orbiter Map"""
```

Comment:
Check here. """Tests for EUI Solar Orbiter Map""" > """Tests for WISPR Parker Solar Probe Map""" ?